tests/test-upgrade-and-update/02-update.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+    - script: |
+        kubectl kudo update --instance spark \
+          --namespace=$NAMESPACE \
+          --instance=spark \
+          --wait \
+          -p enableHistoryServer=true \
+          -p historyServerFsLogDirectory=s3a://$AWS_BUCKET_NAME/$(cat .tmp/s3_bucket_path)
```

Comment:
Can we use a local folder here? The current approach seems to give false-negative results given the [following error in CI](https://teamcity.mesosphere.io/buildConfiguration/Frameworks_DataServices_Kudo_Spark_Pr_SparkPrKonvoyKudoV0100/3084769?buildTab=log&focusLine=16417&linesState=16417):
```
logger.go:42: 12:39:14 | test-upgrade-and-update/2-update | cat: .tmp/s3_bucket_path: No such file or directory

```

Reply:
changed to the local folder